flask_meld/component.py

```diff
@@ -231,10 +234,17 @@ def _set_values(self, soup, context_variables):
         """
         Set the value on model fields
         """
-        for element in soup.find_all(attrs={'meld:model': True}):
-            element.attrs["value"] = context_variables[
-                element.attrs["meld:model"]
+        for element in soup.select("input,select,textarea"):
+            model_attrs = [
+                attr for attr in element.attrs.keys() if attr.startswith("meld:model")
             ]
```

Comment:
Looks like there is no option to search for attributes which start with a string, so I had to go through all of them. To reduce the load I limited the selected elements to the common tag types of `meld:model`

Reply:
Ah yes, I was finding this same issue. This solution will work well, I like that you are selecting only the input, select, and textarea fields.